src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Tagging {
+    private ArrayList<String> tagList;
+
+    public Tagging() {
+        tagList = new ArrayList<String>();
+    }
+
+    /**
+     * records whenever a person has a tag added
+     * 
+     * @param tag
+     *            tag to be added
+     * @param person
+     *            person who has the tag added to
+     */
+    public void addTag(String tag, ReadOnlyPerson person) {
+        tagList.add("+ " + person.getName().toString() + " [" + tag + "]");
```

Comment:
After storing as a string, you can no longer retrieve and process these information without having to parse the strings.